code/fmri/data/reports-spec.yml

```diff
@@ -0,0 +1,38 @@
+package: funconn
```

Comment:
Not sure this will work, have you tested?

Reply:
Yes it works

Reply:
Here is the group report generated:
![image](https://github.com/TheAxonLab/hcph-sops/assets/77437752/42ffd268-db0d-4d48-a59e-d7389f6c3221)

Reply:
Wow super cool

Reply:
> In the hcph-sops as an example of report?

Of course, that'd be great!